torchvision/datasets/_optical_flow.py

```diff
@@ -208,6 +209,72 @@ def _read_flow(self, file_name):
         return _read_16bits_png_with_flow_and_valid_mask(file_name)
 
 
+class FlyingChairs(FlowDataset):
+    """`FlyingChairs <https://lmb.informatik.uni-freiburg.de/resources/datasets/FlyingChairs.en.html#flyingchairs>`_ Dataset for optical flow.
+
+    You will also need to download the FlyingChairs_train_val.txt file from the dataset page.
```

Comment:
Is not providing download functionality for these datasets intentional?

Reply:
For this one specifically and for Sintel, I think we could provide download functionalities. I'm just leaving this for another potential PR

For the others it's not as easy because e.g. FlyingThings3D can only be downloaded via bittorrent, and for Kitti one need to register on the website first (in theory...).